Formula/sdlpop.rb

```diff
@@ -41,3 +44,23 @@ def caveats
     EOS
   end
 end
+
+__END__
+diff --git a/src/Makefile b/src/Makefile
+index f7181ef..a4deeb2 100644
+--- a/src/Makefile
++++ b/src/Makefile
+@@ -12,14 +12,8 @@ OS      := $(shell uname)
+ CPPFLAGS += -Wall -D_GNU_SOURCE=1
+ CFLAGS += -std=gnu99 -O2
+ 
+-ifeq ($(OS),Darwin)
+-LIBS := $(shell sdl2-config --libs) -lSDL2_image
+-CFLAGS += -I/opt/local/include
+-CPPFLAGS += -D_THREAD_SAFE -DOSX
```

Comment:
I think we might want to keep these `CPPFLAGS`.

This also looks like a patch that will never get upstreamed. We should turn it into one that can be.

Reply:
`CFLAGS += $(shell pkg-config --cflags sdl2 SDL2_image)` will add `-D_THREAD_SAFE` to `CFLAGS`. Not sure if `-DOSX` will impact build.

Assuming MacPorts properly includes the `pkg-config` files for SDL2, I think the upstream should be able to do something similar to this patch.

Here is output of `pkg-config` on my system:
```console
❯ pkg-config --cflags sdl2 SDL2_image
-D_THREAD_SAFE -I/opt/homebrew/include/SDL2

❯ pkg-config --libs sdl2 SDL2_image
-L/opt/homebrew/lib -lSDL2_image -lSDL2
```